boards/arm/stm32l562e_dk/stm32l562e_dk_ns.dts

```diff
@@ -18,11 +18,35 @@
 		zephyr,console = &usart1;
 		zephyr,shell-uart = &usart1;
 		zephyr,sram = &sram0;
-		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot1_partition;
 	};
 
 	aliases {
 		led0 = &green_led_10;
 		sw0 = &user_button;
 	};
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00019000>;
+			read-only;
+		};
+		/* Secure image primary slot */
+		slot0_partition: partition@19000 {
+			label = "image-0";
+			reg = <0x00019000 0x0003C000>;
+		};
+		/* Non-secure image primary slot */
+		slot1_partition: partition@55000 {
+			label = "image-1";
+			reg = <0x00055000 0x0002A000>;
```

Comment:
0x2A000 could be written with DT_SIZE_K(x) format

Reply:
Sure, added as request. I introduced the storage partition.